Update adaptive-cloudwatch-polling.md (#37854)
DataDog/documentation: content/en/integrations/guide/adaptive-cloudwatch-polling.md

Reviewer: we should explicitly mention that monitored metrics always stay active

---
title: Adaptive Polling of CloudWatch Metrics

> private: false

Reviewer 2: You can just delete this line instead of changing to false (pages are public by default)

> private: false

Author: ```suggestion
```

## Considerations

> * Metrics used in Datadog monitors will always be considered active 

Reviewer 2: ```suggestion
* Metrics used in Datadog monitors are always considered active.
```

> * Custom metrics are not included in apative polling  

Reviewer 2: ```suggestion
* Custom metrics are not included in adaptive polling.
```

### Multiple AWS accounts

Adaptive Polling applies across all AWS accounts monitored by Datadog.

> ### What happens when a slowed metric is queried?

Reviewer 2: Is a slowed metric the same as an unqueried metric? If not maybe add a definition to the term "slowed" metric on first use ("a metric that has been throttled due to being unqueried" ?)

Author: Good point -- they are the same. I think we should stick to calling it a slowed metric as it is a bit more logical

> As soon as the metric is queried by any surface (dashboard, monitor, notebook, Metrics Explorer, API query, etc.), Datadog will return that metric to the active 10-minute polling frequency. Note there is a short operational lag: it takes a few minutes between when the query happens and when re-polling of data occurs.

Reviewer 2: ```suggestion
As soon as the metric is queried by any surface (such as a dashboard, monitor, notebook, Metrics Explorer, or API query), Datadog returns that metric to the active 10-minute polling frequency.

**Note:** There is a short operational lag of a few minutes between when the query happens and when re-polling of data occurs.
```

> For example, a customer's metrics may have been crawled at 1:00pm, and then at 1:02pm a dashboard is viewed that contains a slowed metric. This slowed metric will return to active but will not refresh until the next metric crawl, 8 mins later at 1:10pm.  

Author: ```suggestion
For example, a customer's metrics may have been crawled at 1:00 p.m, and then at 1:02 p.m a dashboard is viewed that contains a slowed metric. This slowed metric will return to active but will not refresh until the next metric crawl, 8 mins later at 1:10 p.m.  
```

> For example, a customer's metrics may have been crawled at 1:00 p.m, and then at 1:02 p.m a dashboard is viewed that contains a slowed metric. This slowed metric will return to active but will not refresh until the next metric crawl, 8 mins later at 1:10 p.m.  

Reviewer 2: ```suggestion

For example, if metrics were last polled at 1:00 p.m. and a dashboard containing a slowed metric is opened at 1:02 p.m., the metric returns to active status but does not refresh until the next poll cycle, 8 minutes later at 1:10 p.m.  
```

> ### How do I enable or opt out of Adaptive Polling?

Author: ```suggestion
### How do you enable this or opt out of Adaptive Polling?
```

> All Datadog orgs and AWS accounts are automatically enrolled in adaptive polling. To opt out, please make a request with [Datadog support](https://www.datadoghq.com/support/).

Author: ```suggestion
All Datadog orgs and AWS accounts are automatically enrolled in adaptive polling. To opt out, make a request with [Datadog support](https://www.datadoghq.com/support/).
```

> All Datadog orgs and AWS accounts are automatically enrolled in adaptive polling. To opt out, make a request with [Datadog support](https://www.datadoghq.com/support/).

Reviewer 2: ```suggestion
All Datadog organizations and AWS accounts are automatically enrolled in adaptive polling. To opt out, contact [Datadog support](https://www.datadoghq.com/support/).
```
